### ISSUE TITLE
`navigation.push` triggers multiple screens simultaneously

### ISSUE THREAD

**Issue by wli** [exponentjs/ex-navigation#52]
Here's what the log looks like:
```
1:11:35 PM nav question
1:11:35 PM Analytics push question
1:11:35 PM QuestionScreen willMount
1:11:37 PM pop question
1:11:39 PM nav conversation
1:11:39 PM Analytics push conversation
1:11:39 PM ConversationScreen willMount
1:11:41 PM pop conversation
1:11:45 PM nav question
1:11:45 PM Analytics push question
1:11:45 PM ConversationScreen willMount
1:11:45 PM QuestionScreen willMount
```

- `nav conversation` is logged right before `navigator.push` is called
- `ConversationScreen willMount` is logged inside the `componentWillMount` function
- `Analytics push question` is logged via redux hooks into `EX_NAVIGATION.PUSH`
- timestamps confirm when these happen too

Seems pretty reproducible: push A, pop, push B, pop, push A (which triggers A and B simultaneously)

@skevy @brentvatne

**Comment by skevy** [comment]
Can you give it a try now @wli after this commit is published to npm? https://github.com/exponentjs/ex-navigation/commit/ce10f6501f83066f56cec853ecbdc8482847e6fc

**Comment by skevy** [comment]
published @ 1.5.24

**Comment by wli** [comment]
Sorry I never followed up, this is indeed fixed for me. Thanks!
